### ISSUE TITLE
What does remark 3.3 mean?

### ISSUE THREAD

**Issue by pabryan** [pabryan/rough_metrics_heat_kernel_regularity#10]
- What are $s$ and $\theta$ supposed to be?
- The claim that solution is often formulated to have initial smooth, compactly supported data is not clear. Is that really what is taken often? Usually, something much weaker is taken because the restriction to smooth compactly supported data is too restrictive and suitable theories exist for Lipschitz initial data, Holder continuous initial data and various completions of these spaces with respect to Holder norms such as the little Holder spaces. Classically, the heat equation instantly smooths and we do obtain $C^\infty$ on $t \in (0, \infty)$ starting with such initial data. Also, one can typically relax the compact support to controlled growth at infinity such as sub-exponential which is also classically known for almost a century now.

**Comment by lashputin** [comment]
Right, so I should say that in defining a solution, authors typically only consider smooth solutions. This is true in the smooth setting, a posteriori, but it's a bit sneaky that they ask for this before hand. I actually have no clue what I wrote by s and \theta, but I guess at the end of the day, since H^{1}_0 = H^1,  I think I was tyring to say that \Delta_{g, W} is uniquely determined.
